README.md

```diff
@@ -14,15 +31,19 @@ While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
```

Comment:
"While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great basis to build an ORM. It is also useful if you don't want to commit to a specific database, allowing easy switch between databases."

To me neither of the above sentences are clear. How about "Whilst Swift-Kuery is not an Object-Relational Mapping (ORM), it provides an ideal base on which to build an ORM."... are we not going to go on to say here to at least provide a link to Swift-Kuery-ORM?

For the other sentence...
"If you don't want to commit to a specific database, Swift-Kuery's abstraction layer allows easy switching between different database providers."